app/views/indices/_form.html.haml

```diff
@@ -20,4 +21,6 @@
             .tab-pane{id: dom_id(category), class: category == first_category ? 'active' : '' }
               = render partial: 'category', collection: category.children, locals: {f: f}
     .panel-footer.text-right
-      = f.submit class: 'btn btn-primary'
+      .pull-left
+        Version: #{@index.awrvi_version.name}
+      = f.submit 'Save AWRVI Dashboard', class: 'btn btn-primary'
```

Comment:
👎  on the use of dashboard here.  I really don't like that term.  'Assessment' or something similar might be better.